json/src/spec/authority_round.rs

```diff
@@ -49,6 +49,9 @@ pub struct AuthorityRoundParams {
 	/// Block from which monotonic steps start.
 	#[serde(rename="validateStepTransition")]
 	pub validate_step_transition: Option<Uint>,
+	/// Whether transitions should be immediate.
+	#[serde(rename="immediateTransitions")]
+	pub immediate_transitions: Option<bool>,
```

Comment:
Shouldn't that be only relevant to `ValidatorSet`s that do transitions (unlike `SimpleList`)?